pkgs/development/haskell-modules/configuration-ghcjs.nix

```diff
@@ -4,13 +4,21 @@
 # in the OTHER PACKAGES section.
 { pkgs }:
 
+with import ./lib.nix { inherit pkgs; };
+
 let
   removeLibraryHaskellDepends = pnames: depends:
     builtins.filter (e: !(builtins.elem (e.pname or "") pnames)) depends;
+  addSetupDependGlobally = excludes: pset: dep:
```

Comment:
Wouldn't it be easier to wrap the `mkDerivation` function in the package set to add those dependencies? That would be one override, rather than one override per package in the whole set.